dataprofiler/tests/profilers/test_float_options.py

```diff
@@ -0,0 +1,26 @@
+from dataprofiler.profilers.profiler_options import FloatOptions
+from dataprofiler.tests.profilers.test_numerical_options import TestNumericalOptions
+
+
+class TestFloatOptions(TestNumericalOptions):
+    
+    option_class = FloatOptions    
+    keys = ["min", "max", "sum", "variance", "histogram_and_quantiles", "precision"]
```

Comment:
i think you could do keys = TestNumericalOptions.keys + ["precision"]
or do you think that becomes unclear?